lighthouse-core/report/styles/report.css

```diff
@@ -389,6 +380,15 @@ body {
   width: calc(100vw - var(--report-menu-width));
 }
 
```

Comment:
Container for bottom panel.
Config panel need some CSS variables here to calculate it's width. This container removes the dependency between config panel and the main CSS.